### ISSUE TITLE
including invalid transcripts in training data

### ISSUE THREAD

**Issue by sparthib** [GoekeLab/bambu#461]
Hi, I have a question about how XGboost is used to train TPS prediction for read classes. I can see how true transcripts are readily available from a reference annotation (y_ij = 1) ? But it is unclear to me how invalid transcript observations are input for training (y_ij = 0) ?

Thanks,
Sowmya

**Comment by cying111** [comment]
Hi @sparthib ,

As described in our paper, TPS prediction for all RCs is performed using a supervised machine learning algorithm. During training, the labels for these RCs are determined based on whether their intron junctions align exactly with those of annotated transcripts—in other words, only annotated reference transcripts are used for labeling during training. For more details, please check out our paper [here](https://www.nature.com/articles/s41592-023-01908-w).

I hope this clarifies your question! Let me know if you need any further clarification.

Thank you
Warm regards,
Ying
